codice/ddf
# DDF-4186 Modifying ACL to incorporate R/W permissions on Metacards

#### What does this PR do?
Adds an additional level of granularity to the ACL data model to incorporate read/write perms. 
#### Who is reviewing it? 

#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Ask 2 committers to review/merge the PR and tag them here.

@bdeining 
@djblue 
@middlets719 
@cantstoptheunk 
@Bdthomson 
@adimka 
@gordocanchola 


#### How should this be tested?
Ensure no regressions have occurred for _sharing_ as a whole in search forms, workspaces.

Ensure that the forms can't be modified once shared unless you're an owner or access individual. 

#### Any background context you want to provide?
#### What are the relevant tickets?
[](https://codice.atlassian.net/browse/)
#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
